### ISSUE TITLE
Responsivity

### ISSUE THREAD

**Issue by erikasinkovics** [boranzaza/legato#91]
We could spend some time working on making the app responsive (for laptop and mobile).

**Comment by boranzaza** [comment]
I think this should be a priority, so we can at least share the link to the website with people!

**Comment by erikasinkovics** [comment]
Definitely! It would be very useful to just take out your phone and show them the app for e.g.
